pkgs/by-name/ea/earbuds/package.nix

```diff
@@ -0,0 +1,53 @@
+{
+  lib,
+  stdenv,
+  rustPlatform,
+  fetchFromGitHub,
+  installShellFiles,
+  pkg-config,
+  dbus,
+  libpulseaudio,
+  bluez,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "earbuds";
+  version = "0.1.9";
+
+  src = fetchFromGitHub {
+    owner = "JojiiOfficial";
+    repo = "LiveBudsCli";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-A4XQiJrk4ehb6+935L2JFOeAhUJ7bdukV5mL0Jxn0sQ=";
+  };
+  cargoHash = "sha256-FFZnYsqoG2Fq1K2D0ElwTPXgp9F1MyBj27EOAALfXaM=";
+
+  nativeBuildInputs = [
+    pkg-config
+    installShellFiles
+  ];
+
+  buildInputs = [
+    dbus
+    libpulseaudio
+    bluez
+  ];
+
+  doCheck = false;
+
+  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
+    installShellCompletion --cmd earbuds \
+      --bash <($out/bin/earbuds --generate bash) \
+      --fish <($out/bin/earbuds --generate fish) \
+      --zsh <($out/bin/earbuds --generate zsh)
+  '';
+
+  meta = {
+    description = "A free CLI tool to control your Galaxy Buds";
```

Comment:
```suggestion
    description = "Free CLI tool to control your Galaxy Buds";
```

Reply:
For reference: https://nixos.org/manual/nixpkgs/unstable/#var-meta-description